src/server/pps/server/api_server.go

```diff
@@ -999,7 +999,10 @@ func (a *apiServer) ListDatum(request *pps.ListDatumRequest, server pps.API_List
 	// TODO: Auth?
 	if request.Input != nil {
 		return a.listDatumInput(server.Context(), request.Input, func(meta *datum.Meta) error {
-			return server.Send(convertDatumMetaToInfo(meta))
+			meta.State = datum.State_UNPROCESSED
+			di := convertDatumMetaToInfo(meta)
+			di.Datum.ID = ""
```

Comment:
leaving the Datum ID blank restores the behavior in 1.x. This seems like reasonable behavior to me. I could see value exposing it but in fact this ID is different from that which will get created when live datums are instantiated. I'm still figuring out why that is